AppCenterDataStorage/AppCenterDataStorage/Internal/Client/MSCoreDataClient.m

```diff
@@ -0,0 +1,235 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+#import "MSCoreDataClient.h"
+#import "MSDataStorageConstants.h"
+#import "MSDataStore.h"
+#import "MSDataStoreErrors.h"
+#import "MSDataStoreInternal.h"
+#import "MSDocumentUtils.h"
+#import "MSDocumentWrapperInternal.h"
+#import "MSLogger.h"
+#import "MSTokenExchange.h"
+#import "MSTokensResponse.h"
+
+@implementation MSCoreDataClient : NSObject
+
+- (instancetype)initWithDocumentStore:(id<MSDocumentStore>)documentStore {
+  self = [super init];
+  if (self) {
+    _documentStore = documentStore;
+    _reachability = [MS_Reachability reachabilityForInternetConnection];
+  }
+  return self;
+}
+
+- (void)performCoreOperation:(NSString *_Nullable)operation
```

Comment:
Can you remove some of the comments that describe only a single line of code? I'm finding it a bit hard to read this. Just a nitpick though.